webapp/src/components/Meeting.tsx

```diff
@@ -25,6 +34,9 @@ const Meeting = ({ meetingId }: MeetingProps) => {
     path: `/meetings/${meetingId}`,
     sendCredentials: true,
   });
+  useEffect(() => {
+    meeting && setMeetingNotes(meeting.meeting_notes);
```

Comment:
Why do we have to set Meeting Notes when `meeting` updates?  Doesn't the server push newly created notes to the client when added?